src/ui/pages/SendTransaction/SendTransaction.tsx

```diff
@@ -863,3 +868,228 @@ export function SendTransaction() {
+    invariant(windowId, 'windowId get-parameter is required');
+    invariant(res.solana, 'Solana response is expected');
+    windowPort.confirm(windowId, res.solana);
+    if (next) {
+      navigate(next);
+    }
+  }
+
+  const handleReject = () => {
+    const windowId = params.get('windowId');
+    invariant(windowId, 'windowId get-parameter is required');
+    windowPort.reject(windowId);
+    if (next) {
+      navigate(next);
+    }
+  };
+
+  const addressAction = useMemo(
+    () => solanaTransactionToAddressAction(transaction, wallet.address),
+    [transaction, wallet.address]
+  );
+
+  const { mutate: sendTransaction, ...sendTransactionMutation } = useMutation({
+    mutationFn: async () => {
+      invariant(sendTxBtnRef.current, 'SignTransactionButton not found');
+      return sendTxBtnRef.current.sendTransaction({
+        solTransaction: transaction,
+        // TODO: How to avoid needing to pass this?
+        transaction: undefined,
+        chain: 'solana',
+        feeValueCommon: null,
+        initiator: origin,
+        clientScope: clientScope || 'External Dapp',
+        method: normalizeMethod(method),
+        addressAction,
+      });
+      // throw new Error('to implement');
+    },
+    onMutate: () => 'sendTransaction',
+    onSuccess: (tx) => handleSentTransaction(tx),
+  });
+
+  const { networks } = useNetworks();
+  if (!networks) {
+    return null;
+  }
+  console.log({ addressAction });
+
+  return (
+    <Background backgroundKind="white">
+      <NavigationTitle title={null} documentTitle="Send Transaction" />
+      <PageColumn
+        // different surface color on backgroundKind="white"
+        style={{
+          ['--surface-background-color' as string]: 'var(--neutral-100)',
+        }}
+      >
+        {addressAction ? (
+          <SolDefaultView
+            wallet={wallet}
+            addressAction={addressAction}
+            origin={origin}
+            networks={networks}
+          />
+        ) : null}
+        <Spacer height={16} />
+        {signOnlyFlow ? 'Must sign only' : null}
+      </PageColumn>
+      <PageStickyFooter>
+        <Spacer height={16} />
+        <VStack style={{ textAlign: 'center' }} gap={8}>
+          {sendTransactionMutation.isError ? (
+            <UIText kind="body/regular" color="var(--negative-500)">
+              {txErrorToMessage(sendTransactionMutation.error)}
+            </UIText>
+          ) : null}
+          <div
+            style={{ display: 'grid', gridTemplateColumns: '1fr 1fr', gap: 8 }}
+          >
+            <Button
+              ref={focusNode}
+              kind="regular"
+              type="button"
+              onClick={handleReject}
+            >
+              Cancel
+            </Button>
+            {preferences ? (
+              <SignTransactionButton
+                // TODO: set loading state when {sendTransactionMutation.isLoading}
+                // (important for paymaster flow)
+                wallet={wallet}
```

Comment:
Will paymaster be supported in Solana? O_O

Reply:
Not sure, probably not

Reply:
I think this is a copied comment